src/app/views/query-runner/request/permissions/TabList.tsx

```diff
@@ -72,8 +72,14 @@ const TabList = ({ columns, classes, renderItemColumn, renderDetailsHeader, maxH
         <FormattedMessage id='Permissions' />&nbsp;({filteredPermissions.length})
       </Label>
       <Label className={classes.permissionText}>
-        {!tokenPresent && <FormattedMessage id={permissionModeType ? 'sign in to consent to permissions' : 'sign in to consent to application permissions'} />}
-        {tokenPresent && <FormattedMessage id={permissionModeType ? 'permissions required to run the query' : 'application permissions required to run the query'} />}
+        {!tokenPresent
+          && <FormattedMessage id={permissionModeType
+            ? 'sign in to consent to permissions'
```

Comment:
shouldn't these strings also pass to `translateMessage`?

Reply:
The other `FormattedMessage`s don't use the `translateMessage` so I will leave as is